### PR TITLE
Add CurrentUserProfile UI

### LINKED ISSUE #88: Create view for habit list

### PR DESCRIPTION
**Pull request by Docker-J**
## Description of Changes
Add Current User Profile Fragment UI with habit customlist

TODO: Background

## Screenshots
Screenshot_1635993712

## Checklist

- [ ] Automated tests
- [x] Screenshots included (if necessary)
- [x] Code is well commented

Closes #88

### CODE REVIEW

File: src/app/src/main/java/com/cmput301f21t09/budgetprojectname/UserHabitCustomList.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ArrayAdapter;
+import android.widget.TextView;
+
+import androidx.annotation.Nullable;
+
+import java.util.ArrayList;
+
+public class UserHabitCustomList extends ArrayAdapter<HabitModel> {
```
Comment (MacMalainey) [inline src/app/src/main/java/com/cmput301f21t09/budgetprojectname/UserHabitCustomList.java:14]: Please add comments

**Review by MacMalainey — CHANGES_REQUESTED** [review]
Add JavaDoc to

**Review by loravocado — APPROVED** [review]
I love it! LGTM ❤️

**Review by cheetiantey — APPROVED** [review]
LGTM!

**Review by MacMalainey — APPROVED** [review]

**Pull request merged**